osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -2717,6 +2717,36 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/crescent/images/cre.svg"
       },
       "coingecko_id": "crescent-network"
+    },
+    {
+      "description": "The native token of Oraichain",
+      "denom_units": [
+        {
+          "denom": "ibc/161D7D62BAB3B9C39003334F1671208F43C06B643CC9EDBBE82B64793C857F1D",
+          "exponent": 0,
+          "aliases": [
+            "orai"
+          ]
+        },
+        {
+          "denom": "orai",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/161D7D62BAB3B9C39003334F1671208F43C06B643CC9EDBBE82B64793C857F1D",
+      "name": "Oraichain",
+      "display": "orai",
+      "symbol": "ORAI",
+      "ibc": {
+        "source_channel": "channel-13",
+        "dst_channel": "channel-216",
+        "source_denom": "orai"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/orai.png",
```

Comment:
please point the URI's to the chain registry. You can use:
```
"logo_URIs": {
        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.png",
        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.svg"
      },
```

Reply:
https://registry.ping.pub/oraichain/assetlist.json this is the official docs for osmosis